### ISSUE TITLE
ImportError: cannot import name viewkeys

### ISSUE THREAD

**Issue by madelinezec** [html5lib/html5lib-python#336]
I have python code that uses mechanize to login to a website. With six==1.4.1 and six==1.10.0, I get this error:

```
Traceback (most recent call last):
  File "scrapeRecipe.py", line 31, in <module>
    br.select_form(name="aspnetForm")
  File "/Library/Python/2.7/site-packages/mechanize/_mechanize.py", line 619, in select_form
    global_form = self._factory.global_form
  File "/Library/Python/2.7/site-packages/mechanize/_html.py", line 260, in global_form
    self.forms()
  File "/Library/Python/2.7/site-packages/mechanize/_html.py", line 267, in forms
    self._current_forms, self._current_global_form = self._get_forms()
  File "/Library/Python/2.7/site-packages/mechanize/_html.py", line 282, in _get_forms
    if self.root is None:
  File "/Library/Python/2.7/site-packages/mechanize/_html.py", line 247, in root
    response, verify=False))
  File "/Library/Python/2.7/site-packages/mechanize/_html.py", line 145, in content_parser
    from html5lib import parse
  File "/Library/Python/2.7/site-packages/html5lib/__init__.py", line 16, in <module>
    from .html5parser import HTMLParser, parse, parseFragment
  File "/Library/Python/2.7/site-packages/html5lib/html5parser.py", line 2, in <module>
    from six import with_metaclass, viewkeys, PY3
ImportError: cannot import name viewkeys
```
This appears to similar to[ this post](https://github.com/html5lib/html5lib-python/issues/298), except changing the version of six package didn't fix anything.

**Comment by jakearchibald** [comment]
I'm also getting this error when trying to run Bikeshed

**Comment by jakearchibald** [comment]
I "fixed" it by doing

```sh
sudo pip install --ignore-installed six
```

**Comment by yellowdoge** [comment]
I got the same problem and the same solution, thanks @jakearchibald !  (+@tabatkins FYI)

**Comment by Martin-Pitt** [comment]
Thanks @jakearchibald. I had the same problem but when installing Bikeshed for the first time on OSX. Seems to be working now with your solution.

**Comment by tabatkins** [comment]
And I've [updated the Bikeshed docs accordingly](https://github.com/tabatkins/bikeshed/issues/1035). Thanks!

**Comment by Chronial** [comment]
This should already be fixed and is basically a duplicate of #301.

**Comment by willkg** [comment]
@Chronial Awesome--thanks!
